### PR TITLE
Article: Bulma

### PR DESCRIPTION
**Pull request by theflametrooper**
Refer #707

### CODE REVIEW

**Comment by alayek** [comment]
# LGTM :+1: 

@abhisekp this is totally up your alley!

File: Bulma.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Bulma
+
+Bulma is a modern CSS framework based on [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
```
Comment (abhisekp) [inline Bulma.md]: **Later** link to our internal wiki **flexbox** article after the PR https://github.com/FreeCodeCamp/wiki/pull/728 is merged.

```md
[Flexbox](CSS-Flexbox)
```

----
TODO: Later

Comment (abhisekp) [inline Bulma.md]:
```diff
Bulma is a modern CSS framework based on Flexbox
+ and highly customizable using Sass (SCSS).
```

**Comment by abhisekp** [comment]
**File Rename:**
```diff
- Bulma.md
+ CSS-Framework-Bulma.md
```

**Comment by theflametrooper** [comment]
@abhisekp Done!

**Pull request merged**